.github/workflows/packaging.yml

```diff
@@ -3,6 +3,18 @@ name: packaging
 on: [push, pull_request]
 
 jobs:
+  version_check:
+    runs-on: ubuntu-18.04
+    steps:
+      # Required to the version_checking action running.
```

Comment:
Let's put checkout to version_checking then

Reply:
Do you mean something like this:

packaging.yml
```Yml
jobs:
  version_check:
    runs-on: ubuntu-18.04
    steps:
      # # Required to the version_checking action running.
      # - name: Clone the module
      #   uses: actions/checkout@v3

      - name: Run action
        uses: ./.github/actions/version_checking
        with:
          module-name: 'queue'
```

Action.yml
```Yml
runs:
  using: "composite"
  steps:
    # Added
    - name: Clone the module
      uses: actions/checkout@v3

    - name: Setup tarantool
      uses: tarantool/setup-tarantool@v2
      with:
        tarantool-version: '2.10'

...
...
...
```

I tried, but [version_check](https://github.com/tarantool/queue/actions/runs/4343172137/jobs/7584884131#step:2:1):
```
Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/work/queue/queue/.github/actions/version_checking'. 
Did you forget to run actions/checkout before running your local action?
```

Reply:
Oh, I get it now. It should be okay after you extract action to a separate action repo.

Reply:
Now using:

```yml
jobs:
  version-check:
    runs-on: ubuntu-20.04
    steps:
      - name: Run action
        uses: tarantool/actions/check-module-version@igrishnov/gh-204-add-hardcoded-versioning-support
        with:
          module-name: 'queue'
          repository: 'tarantool/queue'
```

The `@igrishnov/gh-204-add-hardcoded-versioning-support` will be deleted

Also I added `needs: version-check` to `package` job of `packaging.yml` and deleted local `action.yml`